### PR TITLE
Use UglifyJS with ES6+ support

### LINKED ISSUE #716: Suppress Uglify’s 32000 characters warning
In a new DoneJS app, I get this warning when I build my app:

```
WARN: Output exceeds 32000 characters
```

This is harmless but disturbing to see when building a new app. I think it’d be nice if we suppressed this warning. I think it’s something we could work around by specifying Uglify’s `max-line-len` option.

### PR DESCRIPTION
**Pull request by m-mujica**
Closes #716 and #740

success-gif

### CODE REVIEW

**Comment by matthewp** [comment]
lgtm!

**Comment by matthewp** [comment]
Shouldn't the require() call change too, since it's a different package?

File: lib/build_types/minify_js.js [hunk 1]
```diff
@@ -18,23 +18,22 @@ exports.async = function(source, options) {
 };
 
 function uglify(source, options) {
+	var UglifyJS = require("uglify-es");
```
Comment (m-mujica) [inline lib/build_types/minify_js.js]: @matthewp do you mean this require?

Comment (matthewp) [inline lib/build_types/minify_js.js]: :disappear:

**Pull request merged**